conf/messages.en

```diff
@@ -77,27 +77,27 @@ amlSupervisor.FinancialConductAuthority = Financial Conduct Authority (FCA)
 amlSupervisor.Other = Other
 amlSupervisor.selectFromList = Enter the name of your professional body or select from the list
 amlSupervisor.error.required = Select an AML supervisor
-amlSupervisor.opb.AssociationOfAccountingTechnicians = Association of Accounting Technicians
```

Comment:
Any particular reason you decided to remove the 'opb'?

Reply:
Yeah, this was so that I could reuse the messages on the Check your answers in an easy/generic way, otherwise I would have had to duplicate the content in the messages file just for Check your answers